docs/reference/converters.rst

```diff
@@ -0,0 +1,6 @@
+==========
+Converters
+==========
+
+.. automodule:: zyte_common_items.converters
```

Comment:
Getting `WARNING: py:class reference target not found: web_poet.page_inputs.url._Url` here, but as it's indeed not mentioned in the web-poet docs I guess there is nothing to do?

Reply:
I think that's ok, if it doesn't break CI